src/conditionals/simple_programs_with_decision_points/medium/q1/InvestOpTest.java

```diff
@@ -0,0 +1,37 @@
+package conditionals.simple_programs_with_decision_points.medium.q1;
+
+/*
+Write a program that will ask the user for their investment and double it if it is above $5000
+ */
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.RandomInteger;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.RepeatedTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
+import java.util.Random;
+
+
+public class InvestOpTest extends BaseTest {
+    public Random r = new Random();
+
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("What is your investment in \\$ \\(rounded to the nearest dollar\\)\\? "),
+                new NewLine(),
+                new StringLiteral("Investment \\: \\$"),
```

Comment:
":" might not need extra \\, but I'm not sure myself

Reply:
I think it does, but I will double-check

Reply:
You were right, it does not